src/main/java/core/basesyntax/classes/figures/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.classes.figures;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final int shortBase;
+    private final int longBase;
+    private final int height;
+

```

Comment:
Remove this unnecessary empty line to maintain a clean and organized code structure.